src/Kanbas/Courses/Quizzes/Preview/Question/Common/Header/index.css

```diff
@@ -0,0 +1,15 @@
+.header {
```

Comment:
Lets stay consistent with the class names in CSS and use a `wd-` prefix for custom class names.

Reply:
Also, use wd-quiz-question-header instead of just header, and follow this for other custom classes as well.